cmake/toolchain/linker/mold.cmake

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2014, Ruslan Baratov
+# All rights reserved.
+
+if (DEFINED POLLY_FLAGS_LINKER_MOLD_CMAKE_)
+  return()
+else ()
+  set(POLLY_FLAGS_LINKER_MOLD_CMAKE_ 1)
+endif ()
+
+find_program(MOLD_FOUND mold)
+if(NOT MOLD_FOUND)
+  fatal_error("mold not found")
+endif()
+
+include(${CMAKE_CURRENT_LIST_DIR}/../../add_cache_flag.cmake)
```

Comment:
I think it's better to add our cmake/ dir to CMAKE_MODULE_PATH and then you can use just the file name.

Reply:
We decided to leave it as it is